spec/mongoid/relations/embedded/one_spec.rb

```diff
@@ -1019,4 +1019,18 @@ class << person
       end
     end
   end
+
+  context "when parent validation of child is set to false" do
+
+    let(:building) do
+      building = Building.create
+      building.building_address = BuildingAddress.new
+      building.save
+      building.reload
+    end
+
+    it "parent successfully embeds an invalid child" do
+      expect(building.building_address).to be
```

Comment:
You could do

```ruby
expect(building.building_address).to be_a(BuildingAddress)
```

Generally, you want the tests you are doing to be as explicit as possible. Not just testing 'not nil' or 'true'